mesa/model.py

```diff
@@ -68,6 +70,9 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.current_id = 0
         self.agents_: defaultdict[type, dict] = defaultdict(dict)
 
+        self.steps_: int = 0
+        self.time_: TimeT = 0  # the model's clock
```

Comment:
I would really like this to be `_steps` and `_time`. I know agents currently is the other way around, but the long term goals should be having `_var` as mesa reserved variables.

Reply:
Done.